### PR TITLE
fix: reset password form

### PR DESCRIPTION
**Pull request by KubaBaniak**
This form should be on frontend-side and should send request to my backend, but because I don't have one, i had to hard code url

### CODE REVIEW

File: src/utils/reset-password-form.html [hunk 1]
```diff
@@ -112,7 +112,7 @@ <h2>Password Reset</h2>
             newPassword: newPassword,
           };
 
-          fetch('http://localhost:3000/auth/reset-password', {
+          fetch('http://13.49.21.92:3000/auth/reset-password', {
```
Comment (rad20c) [inline src/utils/reset-password-form.html:115]: this shouldn't be hardcoded - use environment variables

**Review by rad20c — CHANGES_REQUESTED** [review]
check comment

File: src/utils/reset-password-form.html (hunk 1 above)
Comment (rad20c) [reply src/utils/reset-password-form.html:115]: I mean the server IP / domain address of course (+ port)

Comment (rad20c) [reply src/utils/reset-password-form.html:115]: you might also want to include `http://` too, as on the server you should use `https://`, while locally localhost most probably runs on `http://`

Comment (KubaBaniak) [reply src/utils/reset-password-form.html:115]: As I mentioned in description, this is 'raw' html. And i think i cannot pass env variables to it (this form is being hosted using Apache HTTP Server)

**Pull request closed without merging**